pkg/u8g2/patches/0002-u8g2-add-riot-os-interface.patch

```diff
@@ -4,17 +4,17 @@ Date: Wed, 22 Jun 2016 18:04:31 +0200
 Subject: [PATCH 2/3] u8g2: add riot-os interface.
```

Comment:
I'd prefer to keep this patch untouched and introduce another patch that does the adaptation